cinema/urls.py

```diff
@@ -15,6 +17,9 @@
 router.register("cinema_halls", CinemaHallViewSet)
 router.register("movies", MovieViewSet)
 router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
+router.register("ticket", TicketViewSet)
```

Comment:
The endpoint for `TicketViewSet` is registered as "ticket". For consistency with other endpoints, consider using the plural form "tickets" unless there's a specific reason to use the singular form.